fiwareskuld/phase/phase0_generateuserlist_alt.py

```diff
@@ -164,6 +260,30 @@ def _get_remaining_trial_time(self, user):
 
         return trial_duration - difference.days
 
+    def _get_remaining_community_time(self, user):
```

Comment:
I do not see the need to create a new method when the only difference is trial_started_at and trial_duration with community_started_at and community_duration.

I just add a new parameter with values trial and community and calculate the other two based on this parameter:
type_user + '_started_at'
type_user + '_duration'